### PR TITLE
mceinject: init at unstable-2013-01-19

### PR DESCRIPTION
**Pull request by arkivm**
###### Motivation for this change
`mce-inject` allows one to inject machine checks on the x86 kernel. This is useful for testing Error detection and Correction (EDAC) [drivers](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/edac) in the Linux kernel.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -888,6 +888,12 @@
     githubId = 1296771;
     name = "Anders Riutta";
   };
+  arkivm = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:891]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Comment (arkivm) [reply maintainers/maintainer-list.nix:891]: Sure. Should I sort it by `maintainer.mname` or by `maintainer.mname.name`? What would you suggest?

Comment (legendofmiracles) [reply maintainers/maintainer-list.nix:891]: The former usually

Comment (arkivm) [reply maintainers/maintainer-list.nix:899]: Done. thanks
